src/devices/usbCamera/ultrapython_ini/ultrapython_hires.ini

```diff
@@ -0,0 +1,10 @@
+device grabberDual
+subdevice usbCamera
+period 71
+capabilities COLOR
+twoCameras false
+name /grabber
+camModel ultrapython
+framerate 1
+honorfps false
+d /dev/media0
```

Comment:
The ini files for running the device are usually specific to the machine on which you will run it, you can put them as example in the documentation of the device (see for example https://github.com/robotology/yarp/blob/0e8c016d10d43307e7b40ba3f7c173286e7d87b0/src/devices/ServerFrameGrabberDual/ServerFrameGrabberDual.h#L105-L141)

Reply:
May I use ann examples folder in the code folder?